integreat_compass/cms/models/offers/offer_version.py

```diff
@@ -113,4 +127,5 @@ class Meta:
         verbose_name = _("offer version")
         verbose_name_plural = _("offer versions")
         default_related_name = "offer_versions"
-        ordering = ["-offer_version_date"]
+        ordering = ["offer", "-created_at"]
+        default_permissions = ("change", "delete", "view")
```

Comment:
```suggestion
        default_permissions = ()
```
Do we even need these permissions at all? Or could we just argue, that we use the offer permissions for offer versions as well?